tests/fixtures.py

```diff
@@ -5,6 +5,13 @@
 from baseframe import _, __
 import baseframe.forms as forms
 
+app1 = Flask(__name__)
+app1.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite://'
+app1.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app2 = Flask(__name__)
+app2.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql:///baseframe_test'
```

Comment:
My development config for PostgreSQL is specified slightly differently (with the username and port). So I borrowed the config from [coaster](https://github.com/hasgeek/coaster/blob/master/tests/test_models.py#L26).